### DISCUSSION TITLE
Stripe multiple accounts

### DISCUSSION THREAD

**Discussion by jaahoo** [pay-rails/pay#901] [Q&A] [answered]
Hi folks,
can I connect multiple Stripe accounts using pay gem? The client has two legal entities and would like to have different payments to each of them. From what I read in the Stripe documentation, you need to create another account.
That would mean additional keys, but only one is entered into the pay gem, right? I'd like someone to advise me if they know how to do this. Thank you.

**Comment by henrydjacob** [comment]
Did you managed to solve this problem? we too have a multi tenant application, each one have separate Stripe account with different keys

**Reply by excid3** [reply, marked as answer]
Use Stripe Connect.
